sass/_b2-support-features.scss

```diff
@@ -0,0 +1,24 @@
+.b2-support-features { 
+  @extend %section-layouts;
+  background-color: $background-A;
+  & .slant {
+    @extend %svg;
+    fill: $background-B;
+    background-color: transparent;
+  }
+}
+.b2-iphone-pic {
+	margin-bottom: 2em;
+}
+
+.b2-feature{
+	margin-left: 29%;
```

Comment:
This has the same issue as the [features list](https://github.com/SEA-Design-Dev/project_week_one/pull/11/files#diff-0f5fc010e73b756e348d75bba990c52bR32)